### ISSUE TITLE
How to add markers in AE

### ISSUE THREAD

**Issue by tianrandesign** [airbnb/lottie-ios#1016]
## Lottie-iOS Issue
Hello! I'm not quite understanding this part in the documentation: 

> Using Markers
> Markers are a way to describe a point in time by a key name. Markers are encoded into animation JSON. By using markers a designer can mark playback points for a developer to use without having to worry about keeping track of animation frames. 

So what type of marker it's referring to? I tried the **layer marker** as said in [Adobe's toturial](https://helpx.adobe.com/after-effects/using/layer-markers-composition-markers.html). But it wasn't exported into JSON file.

## Which Version of Lottie are you using?

Lottie 3.0

## What Platform are you on? 

- [] iOS

## Expected Behavior 

![image](https://user-images.githubusercontent.com/22111745/63642126-9d64ed80-c688-11e9-9c8a-1968427d2499.png)

## Actual Behavior

Marker Not Encoded in JSON file

**Comment by oblogic7** [comment]
I just went through this as well.  Layer markers are not exported, but composition markers are.

Here is how I ended up doing it.  I'm an AE noob, so there may be a better way. 
https://monosnap.com/file/qyCH6SOCr0OGXVEPrMzbNkKeyQzX1L

One other thing I noticed is that it will export new line characters in the marker name if you hit enter like I did in the video.  So be careful about that since Lottie won't match that unless it is included.
